storm-core/src/jvm/org/apache/storm/pacemaker/PacemakerClientHandler.java

```diff
@@ -69,7 +69,7 @@ else if(evm instanceof HBMessage) {
 
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent event) {
-        LOG.error("Connection to pacemaker failed", event.getCause());
+        LOG.error("Connection to pacemaker failed. Trying to reconnect {}", event.getCause().getMessage());
```

Comment:
If we know which exceptions we want to ignore, perhaps just IOExceptions, then we can explicitly blacklist them, and even log them under warn, instead of error.